doc/ops/aws-secrets-manager-setup.md

```
@@ -0,0 +1,3 @@
+# AWS Secrets Manager setup
+
+The AWS secrets setup is described [here](https://opensupplyhub.atlassian.net/wiki/spaces/SD/pages/1436844035/First+stage+transition+to+AWS+Secrets+Manager). It also covers the transition to AWS Secrets Manager.
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)